contracts/feeds/customfeeds/PriceFeedbStablestMATIC.sol

```diff
@@ -0,0 +1,27 @@
+pragma solidity ^0.8.0;
+
+import "../../../interfaces/IPriceFeeds.sol";
+import "../../interfaces/IBalancerVault.sol";
+import "../../interfaces/IBalancerPool.sol";
+
+import "@openzeppelin-4.7.0/token/ERC20/IERC20.sol";
+contract PriceFeedbStablestMATIC {
+    IPriceFeeds internal constant _PRICEFEED = IPriceFeeds(0x600F8E7B10CF6DA18871Ff79e4A61B13caCEd9BC);
```

Comment:
why underscore?

Reply:
internal so uses underscore to differentiate.